docs/FAQ.rst

```diff
@@ -265,9 +265,15 @@ What TCP ports does Avalon use?
   The URL is ``http://localhost:9090/`` or, for Docker,
   ``http://avalon-lmdb:9090/``
 - TCP 5555: ZMQ connections to Avalon Enclave Manager from Avalon Listener.
-  This is used for Avalon singleton enclave workers using Synchronous Mode
+  This is used for Avalon singleton enclave workers using Synchronous Mode.
   The URL is ``tcp://localhost:5555`` or, for Docker,
   ``tcp://avalon-enclave-manager:5555``
+- TCP 7777: ZMQ connections to Avalon Graphene Enclave Manager from
+  Avalon Listener.
```

Comment:
ZMQ socket port used by Avalon Graphene Enclave Manager to communicate with Graphene Python Worker.

Reply:
@manojgop Thanks for the correction.